packages/server/graphql/queries/helpers/fetchAllIntegrations.ts

```diff
@@ -10,15 +13,16 @@ const fetchAllIntegrations = async (
   context: any,
   info: GraphQLResolveInfo
 ) => {
-  const results = await Promise.allSettled([
+  const results = (await Promise.allSettled([
     fetchAtlassianProjects(dataLoader, teamId, userId),
     fetchGitHubRepos(teamId, userId, dataLoader, context, info)
-  ])
-  const allIntegrations = results.flatMap((result) =>
-    result.status === 'fulfilled' ? result.value : []
-  )
+  ])) as PromiseSettledResult<Integration[]>[]
+
+  const allIntegrations = results.flatMap((result) => {
+    return result.status === 'fulfilled' ? result.value : []
+  })
 
-  const getValue = (item) => (item.nameWithOwner || item.projectName).toLowerCase()
+  const getValue = (item) => (item.nameWithOwner || item.name)?.toLowerCase()
```

Comment:
-1 remove here & let's do the sorting in the calling function (see comment above)